lib/sqeduler/service.rb

```diff
@@ -102,7 +102,8 @@ def config_redis_pool
         redis_pool = if config.redis_pool
           config.redis_pool
         else
-          redis = { :namespace => "sqeduler" }.merge(config.redis_hash)
```

Comment:
Right now we expect keys as symbols. Another approach is to symbolize the keys so we can accept keys as symbols or strings, which fixes our problem and removes a constraint. We can take this code from rails: https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/hash/keys.rb#L54